content_resolver/config_manager.py

```
@@ -861,9 +823,7 @@ def get_configs(self):
                     try:
                         document = yaml.safe_load(file)
                     except yaml.YAMLError as err:
-                        raise ConfigError("Error loading a config '{filename}': {err}".format(
-                                    filename=yml_file,
-                                    err=err))
+                        raise ConfigError(f"Error loading a config '{yml_file}': {err}")
```

Ruff (0.16.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)